server.go

```diff
@@ -1,26 +1,34 @@
 package main
 
 import (
-	"gowebapp/routers"
+	"flag"
+	"strings"
+
+	datastructure "github.com/IBM/go-web-app/datastructure"
+	routers "github.com/IBM/go-web-app/routers"
```

Comment:
We also need to import these from inside the applications as gowebapp/routers because users need to be able to make local changes to these seed apps and have the changes picked up in their applications

Reply:
By using the go-modules, the `datastructure "github.com/IBM/go-web-app/datastructure"` point to the internal code. If you add a method to the `datastructure` code folder, you can use it where you want by importing `datastructure "github.com/IBM/go-web-app/datastructure"` and accessing to the `datastructure.YourNewMethod()`. Maybe I miss the point, by I think that is the right way to achieve it. Naturally, the method have to start with the first capital letter in order to be exported to the other packages.